common/src/main/java/com/wynntils/handlers/actionbar/ActionBarHandler.java

```diff
@@ -59,11 +59,15 @@ public void onActionBarUpdate(ChatPacketReceivedEvent.GameInfo event) {
             WynntilsMod.warn("ActionBarHandler pattern failed to match: " + content);
             return;
         }
+        StyledText[] segments = {
```

Comment:
At this point, the patch has really boiled down to just a change in the ACTIONBAR_PATTERN, and this refactoring is not needed (nor helpful imho).